### DISCUSSION TITLE
feel confused when choosing the best fitted model with loglik

### DISCUSSION THREAD

**Discussion by stevenhuyi** [kingaa/pomp#183] [Q&A]
I have some climatic covariates (e.g., temperature, humidity, and precipitation) and time seiries of hand, foot, and mouth disease (HFMD) data available. I intended to test if climatic factors drive the seasonality of  HFMD. Hence, I used mif2 function to fit a SEIR model to the HFMD data. As I would like to test if  lag effects of climatic factors on HFMD exist, zero week lag to four weeks lag were compared via loglik. The results are as follows:
```
loglik	           loglik.se		lag
-1655.636495	0.770260243		0
-1646.292285	0.424285388		1
-1649.543875	3.008350385		2
-1644.713864	0.461131216		3
-1646.354714	0.346375666		4
```
According to the max logik, model of lag 3 is identified as the best fitting model.  when simulating using paramers of lag 3, the result is poor as follows：
<img width="942" alt="lag3" src="https://user-images.githubusercontent.com/39373773/205005799-20fd5e76-12b3-417e-9085-87a0e681badb.png">

While the simulation of lag 0 is visually better as follows:
<img width="960" alt="lag0" src="https://user-images.githubusercontent.com/39373773/205006323-b430bc02-1f9b-4771-b5ef-a5ce5debb655.png">

Hence, I'm really confused why this happened. Any reply will be much appreciated.

Yi

**Comment by kingaa** [comment]
Your question can be rephrased as "Why does the model that looks best to the likelihood not look best to me?"  That is&mdash;assuming you have indeed maximized the likelihood&mdash;you are noticing that your "eyeball criterion" disagrees with the model's assessment.

Observe that your eyeball criterion weighs and summarizes the discrepancies between the data and the model.  How does it do this?  What function are you implicitly using to decide whether a given discrepancy is large or small?

One can also ask the same question of the model.  Is it possible to plot the data and the model predictions in such a way that you can see what the model sees?  Obversely, is it possible to formulate a likelihood function that matches your eyeball criterion?  If you did so, what rationale would you give for that likelihood function?

Some suggestions:
1. In envelope plots such as the ones you show above, the eye tends to reduce the model predictions to a uniform distribution, which is not what the model is actually saying.  Moreover, you throw away all information about the model-predicted autocorrelation of trajectories.  As I advised you before, it is wise to plot a small number (10&ndash;20) of representative trajectories, so that you can compare more of their features with those of the data.  To get an idea of what the distribution looks like, consider also plotting a large number of trajectories as semitransparent traces, so that one can see where they are dense and where diffuse.
2. It is usually possible to find a transformation of the data such that, on the scale of this transformation, the variance in the data (according to the model) is roughly independent of the mean.  This is the scale on which vertical displacements of equal magnitude are equally likely.  Therefore, this is the scale on which the model is looking at the data, i.e., the scale on which the eyeball criterion and the likelihood agree.
